engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -589,36 +468,55 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
       }
   }
 
-  ignore should "return good results for a good query" in {
-    Get(s"/workflows/$version/query?status=Succeeded") ~>
+  it should "return good results for a good query" in {
+    val workflowId = WorkflowId.randomId()
+    val runningId = WorkflowId.randomId()
+    val events = Seq(
+      MetadataEvent(MetadataKey(workflowId, None, WorkflowMetadataKeys.Status), MetadataValue("Succeeded")),
+      MetadataEvent(MetadataKey(runningId, None, WorkflowMetadataKeys.Status), MetadataValue("Running"))
+    )
+
+    publishMetadata(events)
+    forceSummary()
```

Comment:
not sure if I agree this isn't "intellectually honest" but I'd be happy to participate in a discussion led by the team about which kinds of tests the teams feels should go where (unit, mocks, integration, etc).  

The way I think about it here is that... these tests don't take very long to run and they definitely test things are aren't currently tested elsewhere, so that's a "good thing" as opposed to not having them right now because we don't yet have a team agreement on testing philosophy (which I do agree would be great to have).